stream-chat-android-ui-components/src/main/res/values/styles.xml

```diff
@@ -35,4 +35,50 @@
         <item name="streamAvatarTextSize">@dimen/stream_avatar_text_size_large</item>
         <item name="streamAvatarOnlineIndicatorEnabled">true</item>
     </style>
-</resources>
\ No newline at end of file
+
+    <!--MessageInputViewNew-->
+    <declare-styleable name="StreamMessageInputView">
```

Comment:
All attributes here should be prefixed with `stream`, like the rest of them in the file.